packages/app-content-pages/README.md

```diff
@@ -19,16 +19,16 @@ cp env-default .env
 
 To get the credentials, go to https://app.contentful.com/spaces/jt90kyhvp0qv/api/keys (login credentials are in Passbolt), and copy the Space ID and Content Delivery API token from "Testing".
 
-### Running in development
+## Running in development
 
-#### Docker
+### Docker
 
-- `docker-compose up` to run a server on http://localhost:3000.
+- `docker-compose up` to run a server on http://localhost:3000. The `--build` flag can be used to build the container. This builds and runs a local image which matches the Jenkins build except for running behind a proxy.
```

Comment:
Should this be "to run a dev server" to be consistent with the project app docs?

Reply:
I’m not sure. I think this runs a production build with `yarn start`. Development server, to me, would imply `yarn dev`.

Reply:
Sorry, I’m completely wrong. This is running a development server with `yarn dev`. I just saw my note, further down, to run the production build with `yarn start`.